packages/cli/src/info/cdk_info.ts

```diff
@@ -0,0 +1,62 @@
+import { execa as _execa } from 'execa';
+import { indent } from './indent.js';
+
+/**
+ * Format CDK information.
+ * @param execa - The execa to run the cdk command.
+ * @returns The cdk doctor output.
+ */
+export const getCdkInfo = async (execa = _execa): Promise<string> => {
+  const cdkDoctorArgs: string[] = ['cdk', 'doctor', '--', ' --no-color'];
+  let output;
+  try {
+    // type-cast the value to string, we know it will be a string given the options
+    output = (
+      await execa('npx', cdkDoctorArgs, {
+        stderr: 'pipe',
+      })
+    ).stderr;
```

Comment:
is output going to stderr ?
noting to stdout?

perhaps we could use `all` to combine streams to single output, see https://github.com/sindresorhus/execa?tab=readme-ov-file#all-2 ?